format/pyrdp/pyrdp.go

```diff
@@ -0,0 +1,146 @@
+// Processes PyRDP replay files
+// https://github.com/GoSecure/pyrdp
+//
+// Copyright (c) 2022-2023 GoSecure Inc.
+// Copyright (c) 2024 Flare Systems
+// Licensed under the MIT License
+//
+// Maintainer: Olivier Bilodeau <olivier.bilodeau@flare.io>
+// Author: Lisandro Ubiedo
+package pyrdp
+
+import (
+	"embed"
+	"time"
+
+	"github.com/wader/fq/format"
+	pyrdp_pdu "github.com/wader/fq/format/pyrdp/pdu"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/interp"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+//go:embed pyrdp.md
+var pyrdpFS embed.FS
+
+func init() {
+	interp.RegisterFormat(
+		format.PYRDP,
+		&decode.Format{
+			Description: "PyRDP Replay Files",
+			DecodeFn:    decodePYRDP,
+		})
+	interp.RegisterFS(pyrdpFS)
+}
+
+const (
+	READ_EXTRA = true
+
+	// PDU Types.
```

Comment:
A bit tempted to move these into the pdu package, then could be `pyrdp_pdu.FAST_PATH_INPUT` etc, possible even rename `pyrdp` to just `pdu` hmm